_docs/reference/config/mixer/template/listentry.md

```diff
@@ -20,6 +20,22 @@ type: markdown
 ListEntry is used to verify the presence/absence of a string
 within a list.
 
+When writing the configuration, the value for the fields associated with this template can either be a
+literal or an [expression](https://istio.io/docs/reference/config/mixer/expression-language.html). Please note that if the datatype of a field is not istio.mixer.v1.config.descriptor.ValueType,
```

Comment:
istio.mixer.v1.config.descriptor.ValueType should probably be in code--same for every instance of this

Reply:
Hi @Rachael-Graham , if you don't feel strongly about it, I would prefer to leave this for now. The documentation is generated from proto files that are checked in inside the mixer repo and given the code freeze, it will be extremely hard now to submit a PR in mixer that just adds some `code` tags.

Let me know, if you strongly feel for making this change, I can manually edit thse md file for the time being and post 0.2 I can edit the protos.

Reply:
Nope sounds good to me--definitely can leave it for now. a very minor change!